extensions/libmodbus/Libmodbus.vcxproj

```diff
@@ -0,0 +1,188 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
is this a derivative of what is in libmodbus or is this one you created entirely on your own? What is the difference between this one and the one that is in their repo?

Reply:
It is not a derivative, I created it.